packages-ts/gauntlet-terra-contracts/src/lib/rdd.ts

```diff
@@ -21,3 +21,33 @@ export const getRDD = (path: string, fileDescription: string = 'RDD'): any => {
     )
   }
 }
+
+export enum CONTRACT_TYPES {
+  PROXY = 'proxies',
+  FLAG = 'flags',
+  ACCESS_CONTROLLER = 'accessControllers',
+  CONTRACT = 'contracts',
+  VALIDATOR = 'validators',
+}
+
+export type RDDContract = {
+  type: CONTRACT_TYPES
+  contract: any
+  address: string
+  description?: string
+}
+
+export const getContractFromRDD = (rdd: any, address: string): RDDContract => {
+  return Object.values(CONTRACT_TYPES).reduce((agg, type) => {
+    const content = rdd[type]?.[address]
+    if (content) {
+      return {
+        type,
+        contract: content,
+        address,
+        ...((type === CONTRACT_TYPES.CONTRACT || type === CONTRACT_TYPES.PROXY) && { description: content.name }),
+      }
+    }
+    return agg
+  }, {} as RDDContract)
+}
```

Comment:
Not sure about moving it to `gauntlet-terra`, is quite contract specific no? 

I imagine this function to change in the future, it can become quite useful. We can update it when we have other use cases for it

Reply:
That's why it needs more thought, and feel free to leave it here s is. 

But IMO high-level ideas like `RDDContract` & `getContractFromRDD` (not talking about this specific impl) is a core thing and needs to be moved to the core level.

Reply:
Moved to `gauntlet-terra`, and agree, this should go into a common place. Opening an issue on it